pages/common/nano.md

```diff
@@ -3,22 +3,26 @@
 > Simple, easy to use command-line text editor. An enhanced, free Pico clone.
 > More information: <https://nano-editor.org>.
 
+- Open a new file in nano:
+
+`nano`
+
 - Open a specific file:
 
 `nano {{path/to/file}}`
 
-- Open a file positioning the cursor at the specified line and column:
+- Open a specific file, positioning the cursor at the specified line and column:
 
 `nano +{{line}},{{column}} {{path/to/file}}`
 
-- Enable smooth scrolling:
+- Open a specific file and enable smooth scrolling:
 
-`nano -S {{filename}}`
+`nano --softwrap {{path/to/file}}`
```

Comment:
This is not what this argument does. From the man page:

```
       -S, --softwrap
              Display over multiple screen rows lines that exceed the screen's
              width.  (You can make this soft-wrapping occur at whitespace in‐
              stead of rudely at the screen's edge, by using also --atblanks.)
              (The old short option, -$, is deprecated.)
```

Reply:
Also it says:
```
Since version 4.0, nano by default:

           • does not automatically hard-wrap lines that become overlong,
           • includes the line below the title bar in the editing area,
           • does linewise (smooth) scrolling.

If you want the old, Pico behavior back, you can use --breaklonglines,
--emptyline, and --jumpyscrolling (or -bej for short).
```
So I think changing the description should be fine e.g. (`- Open a specific file and enable soft wrapping: `).